iphone/cli/commands/_build.js

```diff
@@ -5818,7 +5818,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 					this.logger.info(__('Processing JavaScript files'));
 					const sdkCommonFolder = path.join(this.titaniumSdkPath, 'common', 'Resources');
 
-					async.eachSeries(Object.keys(jsFiles), function (file, next) {
+					async.each(Object.keys(jsFiles), function (file, next) {
```

Comment:
How about using `async.eachLimit()` instead? If you have a lot of JS files, there's a point that you actually are hurting performance. What happens if you cap this at 10? 50? 100?

Reply:
Side note: We have roughly 276+ JS files in our Titanium project:
```
hans$ cloc app/
     444 text files.
     440 unique files.
      10 files ignored.

github.com/AlDanial/cloc v 1.80  T=0.70 s (625.0 files/s, 49668.5 lines/s)
----------------------------------------------------------------------------------
Language                        files          blank        comment           code
----------------------------------------------------------------------------------
JavaScript                        276           4702           3772          19872
XML                                76             19             20           2299
Titanium Style Sheet               58            424            849           1893
JSON                               28              0              0            959
----------------------------------------------------------------------------------
SUM:                              438           5145           4641          25023
----------------------------------------------------------------------------------
```

Reply:
@cb1kenobi So I actually did test with `eachLimit` `25` and `50` values and found that `each` still outperformed it in the test suite with ~1800 JS files (in my initial round of testing, but I omitted the results from here because I didn't see it being useful once I found `each` outperformed it)

Reply:
OK, thanks for testing. I think we're probably fine not limiting today.